### DISCUSSION TITLE
"wormhole v1 tokens are not eligible for transfer"

### DISCUSSION THREAD

**Discussion by jetlej** [wormhole-foundation/wormhole#499] [Developer Q&A] [answered]
I have wETH and wBTC in my Solana wallet from Wormhole V1. How do I get them back into a form that can be moved back to Ethereum via Wormhole V2?

I don't see them showing up on Raydium, nor can I transfer them via Wormhole anymore.

**Comment by quentinadam** [comment, marked as answer]
On the wormholebridge main page there is a link to the version 1 bridge : 

For assets that don't support the migration, the v1 UI can be found at [v1.wormholebridge.com](https://v1.wormholebridge.com/).

You can use the v1 UI to transfer the tokens back to ETH and afterwards use the v2 (current) UI to sent the tokens to Solana.

**Reply by krezdoni** [reply]
I used v1 to transfer eth to metamask, the transfer went through but never arrived in my metamask. 
Any clue on how to find the lost eth?
thanks

**Reply by leoluk** [reply]
@krezdoni this sounds like a known v1 UI bug, please create an issue here including the tx hash: https://github.com/solana-labs/oyster/issues
